hadoop-common-project/hadoop-common/src/test/java/org/apache/hadoop/fs/TestFileUtil.java

```diff
@@ -55,9 +56,11 @@
 public class TestFileUtil {
   private static final Log LOG = LogFactory.getLog(TestFileUtil.class);
 
-  private static final String TEST_ROOT_DIR = System.getProperty(
-      "test.build.data", "/tmp") + "/fu";
+  private static final String TEST_ROOT_DIR =
+      new File(GenericTestUtils.getTestDir(), "fu").getAbsolutePath();
   private static final File TEST_DIR = new File(TEST_ROOT_DIR);
+  private static final String  cacheDir = System.getProperty("test.cache.data",
+      new File(TEST_DIR, "cache").getAbsolutePath());
```

Comment:
* The code creates String by `File.getAbsolutePath()` and then creates File by `new File(String)`. It can be simplified by the following and `TEST_ROOT_DIR` can be removed.
```
 private static final File TEST_DIR = new File(GenericTestUtils.getTestDir(), "fu");
```

* `cacheDir` is unused.